arctic/serialization/numpy_arrays.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import numpy as np
+import numpy.ma as ma
+import pandas as pd
+
+from bson import Binary, SON
+
+from .._compress import compress, decompress
+
+
+class NumpyArrayConverter(object):
+    """
+    Converts a Numpy ndarray to and from PyMongo SON representation:
+
+        {
+         type:        '<i8',          # Numpy dtype (str)
+         length:      20,             # Numpy array length
+         values:      '\x00\x00bb...' # Compressed bytes
```

Comment:
looks like you also have a `mask` which makes this look almost identical to tickstore.